Frends.FTP.DownloadFiles/Frends.FTP.DownloadFiles.Tests/DownloadFilesTests.cs

```diff
@@ -218,5 +218,25 @@ public void DownloadFTPS_CurrentUserHasNoCertificates()
             Assert.AreEqual(typeof(AuthenticationException), ex.InnerExceptions[0].GetType());

```

Comment:
https://github.com/FrendsPlatform/Frends.FTP/blob/87ac90bd423c797107bfa37efe61aa4308fe1dd1/Frends.FTP.DownloadFiles/Frends.FTP.DownloadFiles.Tests/DownloadFilesTests.cs#L89
No need to log stuff to console.